HW_7/task_7_6/task_7_6.py

```diff
@@ -0,0 +1,32 @@
+# функциюНапишите функцию группового переименования файлов. Она должна:
+# * принимать в качестве аргумента желаемое конечное имя файлов.
+# * При переименовании в конце имени добавляется порядковый номер.
+# * принимать в качестве аргумента расширение исходного файла.
+# Переименование должно работать только для этих файлов внутри каталога.
+# * принимать в качестве аргумента расширение конечного файла.
+from pathlib import Path
+
+
+def rename_groupfiles(result_name, extension, folder):
+    if Path(folder).exists():
+        work_directory = Path(folder)
+        for item in enumerate(work_directory.iterdir(), 1):
+            number, orig_path = item
```

Comment:
Почему бы не сделать заодно
```suggestion
        for number, orig_path in enumerate(work_directory.iterdir(), 1):
```